Multisig/UI/Settings/SafeSettingsViewController/SafeSettingsViewController.swift

```diff
@@ -243,7 +243,11 @@ class SafeSettingsViewController: LoadableViewController, UITableViewDelegate, U
             return tableView.basicCell(name: name, indexPath: indexPath)
 
         case Section.RequiredConfirmations.confirmations(let name):
-            return tableView.basicCell(name: name, indexPath: indexPath, withDisclosure: !safe.isReadOnly, canSelect: !safe.isReadOnly)
+            return tableView.basicCell(name: name,
+                                       indexPath: indexPath,
+                                       withDisclosure: !safe.isReadOnly,
+                                       disclosureImage: UIImage(named: "arrow"),
```

Comment:
It seems the chevron is still missing when switching between readOnly and not read only safes. Seems to be a syncing issue.